tests/testthat/test-missingness.R

```diff
@@ -99,3 +99,50 @@ test_that("plot.missingness respects max_char", {
   md <- missingness(data.frame(long_name = NA, longer_still = 1))
   expect_false(isTRUE(all.equal(plot(md, max_char = 7), plot(md))))
 })
+
+test_that("replace.missingness replace blank", {
```

Comment:
Can you test both `data.frame`'s and `tibble`'s explictly? Also, make sure that dataframes work with both factors and strings by using `stringsAsFactors = FALSE` when creating a couple of them.

Reply:
I added a lot more tests to make sure that I tested `data.frames`'s with `stringAsFactors = FALSE`, normal  `data.frames`'s, and `tibble`'s explicity. Let me know if I have done too much or not enough.